### PR TITLE
tools: add man pages

### LINKED ISSUE #145: Write man pages for rtrclient and cli-validator
Package linting tools like lintian complain about our tools not having man pages and of course they are right about that.

### PR DESCRIPTION
**Pull request by mroethke**
This kind of depends on #169 because it assumes the name change is already made.

Fix #145

### CODE REVIEW

File: tools/rpki-rov.1 [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.\" See the file LICENSE in the top level directory for more details.
+.\"
+.\" Website: http://rtrlib.realmv6.org/
+.\"
+.TH "rpki-rov" "1"
+.SH NAME
+rpki-rov - rpki route origin validator
+.SH SYNOPSIS
+.B rpki-rov
+[\fIhost\fR] [\fIport\fR]
```
Comment (waehlisch) [inline tools/rpki-rov.1]: Maybe we should make clear that "host" is the RPKI RTR cache server.

Btw, does the client connect by default to the well-known rpki-rtr TCP/323 port? If yes, we should make this clear and that the port is optional.

File: tools/rpki-rov.1 [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+.SH DESCRIPTION
+\fBrpki-rov\fR is a cli interface for rpki route origin validation.
```
Comment (waehlisch) [inline tools/rpki-rov.1]: s/cli interface for rpki route origin validation/command line interface that implemends RPKI route origin validation/

File: tools/rpki-rov.1 [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+It can be used interactively or via pipes.
+The format is always "\fIIP PREFIXLENGTH ASN\fR" followed by a newline.
```
Comment (waehlisch) [inline tools/rpki-rov.1]: The format of the input parameters is always ...

File: tools/rpki-rov.1 [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+.SH EXAMPLES
+Pipe into stdin
+.PP
+.nf
+.RS
+echo "93.175.146.0 24 12654" | rpki-rov rpki.example.com 8282
+.RE
+.fi
+.SH BUGS
+Does currently not support ssh based rpki server.
```
Comment (waehlisch) [inline tools/rpki-rov.1]: s/ssh based/ssh-based/

File: tools/rtrclient.1 [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+.B ssh
+[\fB\-k\fR]
+[\fB\-p\fR]
+.IR HOST
+.IR PORT
+.IR USERNAME
+.IR PRIVATE_KEY
+[\fIHOST_KEY\fR]
+.SH DESCRIPTION
+\fBrtrclient\fR allows to connect to an rtr server and print prefix and router key updates.
```
Comment (waehlisch) [inline tools/rtrclient.1]: ... prints prefix, origin AS, and router key updates.

File: tools/rtrclient.1 [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+.SH OPTIONS
+\fB-k\fR
+.RS 4
+Print information about router key updates
+.RE
+\fB-p\fR
+.RS 4
+Print information about prefix updates
```
Comment (waehlisch) [inline tools/rtrclient.1]: about prefix and origin AS.

File: tools/rtrclient.1 [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+.RE
+.SH EXAMPLES
+Print prefix updates from a tcp based server
```
Comment (waehlisch) [inline tools/rtrclient.1]: (see above; valid ROA payloads consists of prefix AND origin AS.)

File: tools/rtrclient.1 [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+.PP
+.nf
+.RS
+rtrclient tcp -k rpki.example.com 8282
```
Comment (waehlisch) [inline tools/rtrclient.1]: s/8282/323/

File: tools/rpki-rov.1 (hunk 1 above)
Comment (mroethke) [reply tools/rpki-rov.1]: How about adding "It connects to the rpki server described by \fIhost\fR and \fIport\fR and can be used …"
to the Description?
I believe that aligns well with other man pages.

Comment (waehlisch) [reply tools/rpki-rov.1]: Sounds good.

Comment (mroethke) [reply tools/rpki-rov.1]: done

**Pull request merged**